app/src/screens/OnboardingPages.tsx

```diff
@@ -52,9 +65,33 @@ const EndPage = (onTutorialCompleted: GenericFn, theme: ITheme['OnboardingTheme'
 const StartPages = (theme: ITheme) => {
   const { t } = useTranslation()
   const defaultStyle = createStyles(theme)
+  const [store, dispatch] = useStore()
+  const developerOptionCount = useRef(0)
+  const touchCountToEnableBiometrics = 9
```

Comment:
If this is shared with Settings I think it should be imported from constants. Else we'll need to change it in two places.

Reply:
Oh. Nm. Looks like that would be overly complicated to get one constant exported from Bifold.